doc/source/docs/user_guide/pygeos_to_shapely.rst

```diff
@@ -0,0 +1,79 @@
+Migration from PyGEOS geometry backend to Shapely 2.0
+=====================================================
+
+Since the 0.8 version, GeoPandas includes an experimental support of PyGEOS as an
+alternative geometry backend to Shapely. Recently, PyGEOS codebase was merged into the
+Shapely project and released as part of the Shapely 2.0. GeoPandas will therefore
+deprecate support of the PyGEOS backend and will go forward with Shapely 2.0 as the
+only geometry engine exposing GEOS functionality.
+
+Given the PyGEOS engine is automatically used if the package is installed (only until
+GeoPandas 0.14, then the preference will change), some downstream code may depend on
+PyGEOS geometries being available as underlying data of a ``GeometryArray``.
+
+This guide outlines the migration from the PyGEOS-based code to the Shapely-based code.
+
+Migration period
+----------------
+
+The migration is planned for three releases spanning approximately one year, starting
+with 0.13 released in the second quartile of 2023.
+
+GeoPandas 0.13
+^^^^^^^^^^^^^^
+
+- PyGEOS is still used as a default backend over Shapely (1.8 or 2.0) if installed,
+  with a ``FutureWarning`` warning about upcoming changes.
+
+GeoPandas 0.14
+^^^^^^^^^^^^^^
+
+- The default backend will become Shapely 2.0 and the PyGEOS will be used only
+  if Shapely 1.8 is installed instead of 2.0 or newer. The PyGEOS backend will still be
+  supported, but a user will need to opt in using the environment variable
+  ``USE_PYGEOS`` as explained in the
+  `installation instructions <../getting_started/install>`__.
+
+GeoPandas 1.0
+^^^^^^^^^^^^^
+
+- GeoPandas will remove support of both PyGEOS and Shapely<2.
+
+How to prepare your code for transition
+---------------------------------------
+
+If you don't use PyGEOS explicitly, there nothing to be done as GeoPandas internals will
+take care of the transition. If you use PyGEOS directly and access an array of PyGEOS
+geometries using ``GeoSeries.values.data``, you will need to make some changes to avoid
+code breakage.
+
+The recommended way is using Shapely vectorized operations on the ``GeometryArray``
+instead of accessing the NumPy array of geometries and using PyGEOS/Shapely operations
+on the array.
+
+This is a common pattern used with GeoPandas 0.12, that should now be avoided in new code:
+
+.. code-block:: python
+
+    >>> import pygeos
+    >>> geometries = gdf.geometry.values.data
+    >>> mrr = pygeos.minimum_rotated_rectangle(geometries)
+
+The recommended way of refactoring this code would look like this:
```

Comment:
```suggestion
The recommended way of refactoring this code would look like this (with Geopandas 0.12 or later):
```
Perhaps this is a bit clumsy way of saying it but I think we should probably also explicitly indicate that 0.12 (or 0.13) can be used for migration for the case that someone is upgrading from an older version, e.g. if upgrading from 0.9 move to 0.12/0.13 then do the refactoring to be forwards compatible with 0.14/ 1.0.